renovate.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [":gitSignOff"],
```

Comment:
you can add this line https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/renovate.json#L4
which will create nice dashboard of dependencies like https://github.com/kubevirt/kubevirt-tekton-tasks/issues/496